autoload/handlers.vim

```diff
@@ -120,15 +120,30 @@ def s:processSignaturehelpReply(lspserver: dict<any>, req: dict<any>, reply: dic
   if sig->has_key('parameters') && result->has_key('activeParameter')
     var params_len = sig.parameters->len()
     if params_len > 0 && result.activeParameter < params_len
-      var label = sig.parameters[result.activeParameter].label
+      var label = ''
+      if sig.parameters[result.activeParameter]->has_key('documentation')
+        label = sig.parameters[result.activeParameter].documentation
+      else
+        label = sig.parameters[result.activeParameter].label
+      endif
       hllen = label->len()
       startcol = text->stridx(label)
     endif
   endif
-  var popupID = text->popup_atcursor({moved: 'any'})
-  prop_type_add('signature', {bufnr: popupID->winbufnr(), highlight: 'LineNr'})
-  if hllen > 0
-    prop_add(1, startcol + 1, {bufnr: popupID->winbufnr(), length: hllen, type: 'signature'})
+  if g:LSP_Echo_Signature
+    echon "\r\r"
```

Comment:
Will this result in a "Press Enter to continue" prompt, if the default 'cmdheight' is used?

Reply:
It seem me no (i use vim8). The purpose of the `echo "\r\r"` it clear cmdline when we call `LspShowSignature` several time in a row (the permit to update the highlight of the current parameter).